mypy.ini

```diff
@@ -1,6 +1,15 @@
 [mypy]
+python_version = 3.8
 show_column_numbers = True
+
+warn_return_any = True
+warn_unused_configs = True
```

Comment:
Please use this config - https://github.com/evrone/python-codestyle/blob/master/README.md

Reply:
fixed as we discussed @artinnok